vcf-dumper/vcf-dumper-lib/src/test/java/embl/ebi/variation/eva/vcfdump/BiodataVariantToVariantContextConverterTest.java

```diff
@@ -55,162 +49,206 @@ public static void setUpBeforeClass() throws Exception {
-                    return "AATGTCTGCACTAGCACGTTGA";
-                }
-                return null;
-            }
-        };
-
         // example samples list
         s1s6SampleList = new ArrayList<>();
         for (int i = 1; i <= 6; i++) {
             s1s6SampleList.add("s" + i);
         }
-
-    }
-
-    @After
-    public void tearDown() throws Exception {
-
     }
 
     @Test
-    public void singleStudySNV() throws CellbaseSequenceDownloadError {
-        // create SNV variant
+    public void singleStudySNV(){
+        // create variant
         VariantSource variantSource = createTestVariantSource(STUDY_1);
         String variantLine =  String.join("\t", CHR_1, "1000", "id", "C", "A", "100", "PASS", ".", "GT", "0|0", "0|0", "0|1", "1|1", "1|1", "0|1");
         List<Variant> variants = variantFactory.create(variantSource, variantLine);
         assertEquals(1, variants.size());
 
         // export variant
         BiodataVariantToVariantContextConverter variantConverter =
-                new BiodataVariantToVariantContextConverter(Collections.singletonList(variantSource), null, noSampleNamesConflictSampleNameCorrections);
-        VariantContext variantContext = variantConverter.transform(variants.get(0), null);
+                new BiodataVariantToVariantContextConverter(Collections.singletonList(variantSource),  noSampleNamesConflictSampleNameCorrections);
+        VariantContext variantContext = variantConverter.transform(variants.get(0));
         checkVariantContext(variantContext, CHR_1, 1000, 1000, "C", "A", variants.get(0).getSourceEntries(), false);
     }
 
     @Test
-    public void singleStudyInsertionRegionNull() throws CellbaseSequenceDownloadError {
-        // create SNV variant
+    public void singleStudySingleNucleotideInsertion() {
+        // create variant
         VariantSource variantSource = createTestVariantSource(STUDY_1);
         String variantLine =  String.join("\t", CHR_1, "1100", "id", "T", "TG", "100", "PASS", ".", "GT", "0|0", "0|0", "0|1", "1|1", "1|1", "0|1");
         List<Variant> variants = variantFactory.create(variantSource, variantLine);
         assertEquals(1, variants.size());
 
         // export variant
         BiodataVariantToVariantContextConverter variantConverter =
-                new BiodataVariantToVariantContextConverter(Collections.singletonList(variantSource), cellbaseWSClientStub, noSampleNamesConflictSampleNameCorrections);
-        VariantContext variantContext = variantConverter.transform(variants.get(0), null);
+                new BiodataVariantToVariantContextConverter(Collections.singletonList(variantSource), noSampleNamesConflictSampleNameCorrections);
+        VariantContext variantContext = variantConverter.transform(variants.get(0));
         checkVariantContext(variantContext, CHR_1, 1100, 1100, "T", "TG", variants.get(0).getSourceEntries(), false);
```

Comment:
From our own `Variant` class perspective, the assumptions made for the end coordinate in most of the cases would be wrong (see one of the comments in `BiodataVariantToVariantContextConverter`). Would the `VariantContext` object reject the coordinates as we handle them?